campaigns.py

```diff
@@ -78,9 +78,9 @@ def deploy_banner(application):
     content = env['template_contents']
     put(StringIO.StringIO(content), remote_filename, use_sudo=True, mirror_local_mode=True)
     sudo('chown deploy:deploy %s' % remote_filename)
+    execute(app.restart, application)
```

Comment:
Thats fair. If this were longer lived code i'd be concerned about more applications being added, but thats unlikely given the ESI plans.

Happy either way too, so lets stick with this, unless anyone else has an objection.